hawkbit-repository/hawkbit-repository-test/src/main/java/org/eclipse/hawkbit/repository/test/util/TestdataFactory.java

```diff
@@ -629,18 +596,34 @@ public Target createTarget(final String controllerId) {
         return target;
     }
 
+    /**
+     * @param targetName
+     *            name of the target
+     * @return persisted {@link Target}
+     */
+    public Target createTargetWithName(final String targetName) {
+        final Target target = targetManagement
+                .create(entityFactory.target().create().controllerId(DEFAULT_CONTROLLER_ID).name(targetName));
+        assertThat(target.getCreatedBy()).isNotNull();
```

Comment:
Code duplication with createTarget(). 
You could move the assertion part into a common private class, since this is a helper test class anyway.

Reply:
duplicated code was moved to new method assertTargetProperlyCreated().